sdk_contrib/spring_boot2/README.md

```diff
@@ -0,0 +1,58 @@
+# OpenTelemetry Spring Boot2 Starter
+
+Provides autoconfiguration of the OpenTelemetry SDK in Spring Boot v2.x
+applications. Additional SDK plugins defined elsewhere in the application
+context are injected into the configured and initialized SDK.
+
+## Setup
+
+```groovy
+dependencies {
+  implementation "io.opentelemetry:opentelemetry-api:${otelVersion}"
+  implementation "io.opentelemetry:opentelemetry-sdk:${otelVersion}"
+  implementation "io.opentelemetry:opentelemetry-contrib-spring-boot2:${otelVersion}"
+}
+```
+
+## Configuration Properties
+
+Any or all of these can be specified in `application.properties` or 
+`application.yaml`. If not defined, the default value is used.
+
+| Key                                                                | Default Value | Description                       |
+| :----------------------------------------------------------------- | :------------ | :-------------------------------- |
+| management.opentelemetry.enabled                                   | true          | Enables Spring autoconfiguration of the OpenTelemetry SDK. |
```

Comment:
Is this the right naming convention `management.opentelemetry`? I thought this is usually groupid name  e.g. `io.opentelemetry.*`

Reply:
Properties maintained by the Spring Boot project do not follow this convention. I used management because that is the prefix used by Spring Boot for all actuator-related properties. It maybe is not the best choice. I will look into some more to determine whether this makes sense or not.